actions/release/create/entrypoint/main_test.go

```diff
@@ -133,6 +135,27 @@ func TestEntrypoint(t *testing.T) {
 					t.Fatal(fmt.Sprintf("unknown request: %s", dump))
 				}
 			}))
+
+			file, err := os.CreateTemp("", "output")
+			Expect(err).NotTo(HaveOccurred())
+			githubSetOutputFile = file.Name()
+			Expect(file.Close()).To(Succeed())
+
+			file, err = os.CreateTemp("", "state")
+			Expect(err).NotTo(HaveOccurred())
+			githubSaveStateFile = file.Name()
+			Expect(file.Close()).To(Succeed())
+		})
+
+		it.After(func() {
+			if githubSetOutputFile != "" {
+				Expect(os.Remove(githubSetOutputFile)).To(Succeed())
+				githubSetOutputFile = ""
```

Comment:
I think we can replace this with `os.RemoveAll` and drop the `if` condition. That's analogous to `rm -f` - it will still succeed if the file is not present.